### ISSUE TITLE
Ninki Seiyuu: How to Make a Pop Voice Actress (1777750)

### ISSUE THREAD

**Issue by ipr** [ValveSoftware/Proton#6096]
# Compatibility Report
- Name of the game with compatibility issues:Ninki Seiyuu: How to Make a Pop Voice Actress
- Steam AppID of the game:1777750

## System Information
- GPU: AMD Vega 56/64
- Driver/LLVM version: Mesa 22.0.5
- Kernel version: 5.19.1
- Proton version:7.0-3

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
After starting, game show error dialog (pop-up) and closes after pressing ok.

![Screenshot_20220815_194204](https://user-images.githubusercontent.com/757407/184678867-b3b5ac03-1108-4f1b-8f62-fa58ef0b1d20.png)

## Reproduction
1. start the game
2. select language
3. observe the error

[steam-1777750.log.gz](https://github.com/ValveSoftware/Proton/files/9338614/steam-1777750.log.gz)

**Comment by kisak-valve** [comment]
Hello @ipr, this looks like a quartz / MPEG-1 issue.

From the log:
```
19:44:24 ==== An exception occured at movie.tjs(49)[(function) start], VM ip = 138 ====
19:44:24 -- Disassembled VM code --
19:44:24 #(49)          open(file);
19:44:24 00000138 calld %0, %-2.*7(%-3) // *7 = (string)"open"
19:44:24 -- Register dump --
19:44:24 %-4=(object)(object 0x00CBEDB0:0x00CBEDB0)  %-3=(string)"MintCUBE.mpg"
19:44:24 %-2=(object)(object 0x00B2F460:0x00000000)
19:44:24 %-1=(object)(object 0x18CABE30:0x18CABE30)  %0=(void)  %1=(string)".mpg"
19:44:24 %2=(string)".mpg"  %3=(object)(object 0x00D23730:0x00000000)  %4=(int)1
19:44:24 %5=(object)(object 0x00CBEDB0:0x00CBEDB0)  %6=(void)
19:44:24 ------------------------------------------------------------------------------
```

**Comment by ipr** [comment]
> Hello @ipr, this looks like a quartz / MPEG-1 issue.
> 

Yes, and according to error message similar to issue with  KamiYaba: Destiny on a Dicey Deadline (1856480) #6058 

Both use same engine/technology I think.

**Comment by ipr** [comment]
Problem still exists with Proton 10.1 beta. 

I think upstream Wine already has a fix for the splitter, has that been included in Proton?
